.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
```

Comment:
Quitad boilerplate en los que hayáis generado.

Reply:
Resuelto en [8b99044](https://github.com/javigp2002/LazyFood/pull/43/commits/8b99044ab3380754aac9ccbaaccee843ae5a5546).